packages/x-date-pickers/src/internals/components/wrappers/WrapperProps.ts

```diff
@@ -1,8 +1,9 @@
+import * as React from 'react';
```

Comment:
```suggestion
```

Reply:
My IDE complains when you use JSX without importing React which is not recommended) :angel: 

![image](https://user-images.githubusercontent.com/3309670/191752314-88c485b0-979f-472d-8330-9e7ebff0aed7.png)

Reply:
Ho yes, I missed `React.ElementType` in the interface